apps/web/components/Organization/Wrapper/Themes/UtlendingastofnunTheme/UtlendingastofnunHeader.tsx

```diff
@@ -40,6 +43,14 @@ const UtlendingastofnunHeader: React.FC<
 
   const screenWidth = getScreenWidthString(width)
 
+  const { activeLocale } = useI18n()
+  const logoAltText = n(
```

Comment:
Just in case there isn't a title for the organization perhaps it'd be better to fallback to an empty string instead of potentially concatenating undefined or '' with ' Frontpage'